front/pages/api/w/[wId]/keys/index.ts

```diff
@@ -60,20 +60,38 @@ async function handler(
 
       res.status(200).json({
         keys: keys.map((k) => {
+          // We only display the full secret key for the first 10 minutes after creation.
+          const getSecret = () => {
+            const currentTime = new Date();
+            const createdAt = new Date(k.createdAt);
+            const timeDifference = Math.abs(
+              currentTime.getTime() - createdAt.getTime()
+            );
+            const differenceInMinutes = Math.ceil(timeDifference / (1000 * 60));
+
+            if (differenceInMinutes > 10) {
+              return k.secret.slice(0, 4) + "..." + k.secret.slice(-4);
```

Comment:
There is a function to redact strings `redactString`.

Reply:
<img width="404" alt="Screenshot 2024-04-30 at 12 51 38" src="https://github.com/dust-tt/dust/assets/1189312/cc5c9473-0c56-4a47-a77c-d0efb130dd9b">

Kinda ugly on this use case though, no?

Reply:
Ok, changed redactString to use bullet unicode character (base on Stripe redacting)
<img width="876" alt="Screenshot 2024-04-30 at 12 53 00" src="https://github.com/dust-tt/dust/assets/1189312/a2673d37-9212-414e-8576-d6305b5bdf9b">